var/spack/repos/builtin/packages/adios/package.py

```diff
@@ -12,30 +35,97 @@ class Adios(Package):
     """
 
     homepage = "http://www.olcf.ornl.gov/center-projects/adios/"
-    url = "https://github.com/ornladios/ADIOS/archive/v1.9.0.tar.gz"
+    url      = "https://github.com/ornladios/ADIOS/archive/v1.10.0.tar.gz"
 
-    version('1.9.0', '310ff02388bbaa2b1c1710ee970b5678')
+    # make -j is currently broken
+    parallel = False
```

Comment:
looks like it's not necessary and something went wrong in my last build. will remove